config/smartyadmin.config.inc.php

```diff
@@ -48,7 +48,7 @@
 smartyRegisterFunction($smarty, 'modifier', 'secureReferrer', array('Tools', 'secureReferrer'));
 
 $module_resources['modules'] = _PS_MODULE_DIR_;
-$smarty->registerResource('module', new SmartyResourceModule($module_resources, $isAdmin = true));
+$smarty->registerResource('module', new SmartyResourceModule($module_resources));
```

Comment:
You should keep it, it's needed for some modules

Reply:
No use for our contexts, but nothing says it's not used by modules to determine if the smarty template is renderer in the BO or FO